schemas/chatPoll.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from "zod";
+
+export const BullMQChatJobReturnValue = z.object({
+    chat: z.string(),
+    courseList: z.array(z.any()), // TODO: type this when we declare the interface
+  })
```

Comment:
Here is what is expected from the backend. We still need to define the type/schema we want for courseList contents. My suggestion is the following:

```
interface Course {
  courseCode: string;
  courseName: string;
  courseDescription: string;
  similarityScore: number; // vector match score
  easyRating: number; // from UW flow
  usefulRating: number; // from UW flow
}
```

courseCode, courseName, courseDescription, and similarityScore should be relatively easy to fetch, they are already in chromadb and the similarity can just be the vector score. From our final state we can just pull this from the course list that is returned as matches. easyRating and usefulRating will require us to fetch the UWFlow API, someone needs to look into this